tests/ft.test.ts

```diff
@@ -39,7 +39,7 @@ describe('FT routes', () => {
   test('token not found', async () => {
     const response = await fastify.inject({
       method: 'GET',
-      url: '/ft/SP2SYHR84SDJJDK8M09HFS4KBFXPPCX9H7RZ9YVTS.hello-world',
+      url: '/metadata/v1/ft/SP2SYHR84SDJJDK8M09HFS4KBFXPPCX9H7RZ9YVTS.hello-world',
```

Comment:
We should probably test the non-versioned endpoints as well to ensure they always match the response received from the latest versioned endpoints.

Reply:
Done